enc_temp_folder/ec1feb469841afeb2e47f7b975f/ColouringBookGameMode.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright 1998-2017 Epic Games, Inc. All Rights Reserved.
```

Comment:
I imagine this file is not needed, since you added it to the .gitignore, and I guess the one being used is the regular ColouringBookGameMode.cpp file in Source/. Can you please delete it if this is true? (You must have committed the file before adding it to the .gitignore)

Also, if this is true, I guess you can also remove it from the .gitignore...